backend/tests/api/support/index.js

```diff
@@ -0,0 +1,169 @@
+                .post("/api/supports")
+                .set("accept", "application/json")
+                .send(constructSupportTicket(1))
+                .expect("Content-Type", /json/)
+                .expect(200)
+        });
+
+        it("Authenticated user should post support ticket", async () => {
+            await request(strapi.server.httpServer)
+                .post("/api/supports")
+                .set("accept", "application/json")
+                .set("Authorization", `Bearer ${await jwt(authenticatedUser.id)}`)
+                .send(constructSupportTicket(2))
+                .expect("Content-Type", /json/)
+                .expect(200)
+        })
+
+        it("Admin user should not post support ticket", async () => {
+            await request(strapi.server.httpServer)
+                .post("/api/supports")
+                .set("accept", "application/json")
+                .set("Authorization", `Bearer ${await jwt(adminUser.id)}`)
+                .send(constructSupportTicket(3))
+                .expect("Content-Type", /json/)
+                .expect(500)
+        })
+
+        it("Public user should not find support tickets", async () => {
+            await request(strapi.server.httpServer)
+                .get("/api/supports")
+                .set("accept", "application/json")
+                .expect("Content-Type", /json/)
+                .expect(500)
+        })
+
+        it("Authenticated user should not find support tickets", async () => {
+            await request(strapi.server.httpServer)
+                .get("/api/supports")
+                .set("accept", "application/json")
+                .set("Authorization", `Bearer ${await jwt(authenticatedUser.id)}`)
+                .expect("Content-Type", /json/)
+                .expect(500)
+        })
+
+        it("Admin user should find support tickets", async () => {
+            await request(strapi.server.httpServer)
+                .get("/api/supports")
+                .set("accept", "application/json")
+                .set("Authorization", `Bearer ${await jwt(adminUser.id)}`)
+                .expect("Content-Type", /json/)
+                .expect(200)
+                .then((data) => {
+                    expect(data.body.data.length).toBe(4);
+                });
+        });
+
+        it("Admin user should find a support ticket", async () => {
+            const id = 3
+            const support = constructSupportTicket(id);
+            await request(strapi.server.httpServer)
+                .get("/api/supports/" + id)
+                .set("accept", "application/json")
+                .set("Authorization", `Bearer ${await jwt(adminUser.id)}`)
+                .expect("Content-Type", /json/)
+                .expect(200)
+                .then((data) => {
+                    data = data.body.data;
+                    expect(data.attributes.FullName).toBe(support.data.FullName); //expected name3 received name1 why??
```

Comment:
The expected response is "Test Name3". I would like it to be "Test Name3" but the received response is "Test Name1". I'm not sure why. @doolri1276 @hyunghoseo

Reply:
The title you are giving when posting has NOTHING to do with the actual ID. Public user don't setup id when they post support. The ID is autogenerated and auto increment.

Try this out : using the swagger page to post each supports in the test file in order and from the response check which value you've got for the ID for each support.

Reply:
Got it. I think I was confused because I didn't realize actual ID started with 1 instead of 0. Fixed it now